src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -313,6 +326,45 @@ public void writeBody(PrintWriter w) {
         }
 
         /**
+         * Check if there is a ParameterValue with the same name
+         *
+         */
+        private boolean parameterValueExist (List<ParameterValue> parameterValues, String name) {
+            for(ParameterValue parameterValue : parameterValues) {
+                if (parameterValue.getName().equals(name)) {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        /**
+         * Get the default parameters values from a job
+         *
+         */
+        private List<ParameterValue> getDefaultParametersValues(Job job) {
+            ParametersDefinitionProperty paramDefProp = (ParametersDefinitionProperty) job.getProperty(ParametersDefinitionProperty.class);
```

Comment:
If you checkout the code and try to compile without the cast you will see that you are going to get something like:

```
ERROR] /Users/path/path/plugins/git-plugin/src/main/java/hudson/plugins/git/GitStatus.java:[343,72] incompatible types: hudson.model.JobProperty cannot be converted to hudson.model.ParametersDefinitionProperty
```